.github/workflows/release.yml

```
@@ -65,26 +87,17 @@ jobs:
-          # Copy the built React app to the static directory if needed
-          # This may be redundant if the postbuild script works
-          cp -r renderer/dist/* priv/static/app/ || true
-
-      - name: Update version in mix.exs
-        run: |
-          # Update the version with a valid Elixir version format
-          sed -i "s/version: \"[^\"]*\"/version: \"${{ steps.version.outputs.version }}\"/" mix.exs
-          cat mix.exs
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)